CountingWords.rb

```diff
@@ -0,0 +1,6 @@
+freq = Hash.new(0)
+file = File.open("speech.txt")
+file.read.downcase.scan(/\b[a-z]{3,16}\b/){|word| freq[word] += 1}
```

Comment:
This is the first time I think I've seen someone use `.scan`. me likes it but keep in mind the issues i stated in the other comment. 👍